members/CusA091d190f1518712006.yaml

```diff
@@ -5,18 +5,18 @@ contact_form:
   steps:
     - visit: "https://www.regulations.gov/comment?D=HHS-OCR-2018-0002-0001"
     - find:
-      - selector: div.GIY1LSJGSC form
+      - selector: div form
```

Comment:
are you sure this is the only form element on the page at this point?

Reply:
I think so (?). Screenie below shows what I get when searching for that selector 

![image](https://user-images.githubusercontent.com/8680733/37001634-21bf2992-2095-11e8-9935-db5f439f270a.png)

Reply:
As long as the resulting array has only one element (which in this case it does), then we're good.

Reply:
perfect!